sections/webhooks.md

```diff
@@ -0,0 +1,138 @@
+# Webhooks (New!)
+
+##### Endpoint: `/api/v1/webhooks`
+
+The webhooks feature allows users to register, via the API, a webhook 
+that they can use to receive notifications about key events in their account.
+Currently, webhooks are supported for the following events.
+
+
+| **Event Type** | **Description** |
+| ------------- | --------------- |
+| project.created | fires when a new project is created |
+| project.updated | fires when a project is updated |
+| time.entry.created | fires when a new time entry is created |
+| time.entry.updated | fires when a time entry is updated |
+| user.created | fires when a new time entry is created |
+| user.updated | fires when a user is updated |
+| assignment.created | fires when a new assignment is created |
+| assignment.updated | fires when an assignment is updated |
+
+
+## List Webhooks
+
+```
+GET /api/v1/webhooks
+```
+
+Lists all the webhooks for your organization.
+
+
+#### Example
+
+```
+GET /api/v1/webhooks/
+
+curl 'https://vnext.10000ft.com/api/v1/webhooks?&auth=..'
+```
+
+#### Response
+
+```
```

Comment:
@arathiramani use ```json to get some syntax highlighting around the payload.

Reply:
I tried this but the syntax highlighting looked really weird around some of the placeholder stuff in there. May need to rethink how we author these.